specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2023-11-01/ContentProductPackages.json

```diff
@@ -0,0 +1,231 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/PackageIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/productPackageModel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "productPackageList": {
+      "description": "List available packages.",
+      "properties": {
+        "nextLink": {
+          "description": "URL to fetch the next set of packages.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "value": {
+          "description": "Array of packages.",
+          "items": {
+            "$ref": "#/definitions/productPackageModel"
+          },
+          "type": "array"
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "type": "object"
+    },
+    "productPackageModel": {
+      "allOf": [
+        {
+          "$ref": "../../../common/2.0/types.json#/definitions/ResourceWithEtag"
+        }
+      ],
+      "description": "Represents a Package in Azure Security Insights.",
+      "properties": {
+        "properties": {
+          "description": "package properties",
+          "$ref": "#/definitions/productPackageProperties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object"
+    },
+    "productPackageProperties": {
+      "description": "Describes package properties",
+      "allOf": [
+        {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/packageBaseProperties",
+          "description": "The content id of the package"
+        },
+        {
+          "$ref": "#/definitions/productPackageAdditionalProperties",
+          "description": "The package kind"
+        }
+      ],
+      "required": [
+        "contentId",
+        "contentKind",
+        "version",
+        "displayName"
+      ],
+      "type": "object"
+    },
+    "productPackageAdditionalProperties": {
+      "description": "product package additional properties",
+      "properties": {
+        "installedVersion": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataVersion",
+          "description": "The version of the installed package, null or absent means not installed."
+        },
+        "resourceId": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataContentId",
+          "description": "The metadata resource id."
```

Comment:
Please check [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details) if it is an ARM Resource Id.